Application/Source/Util/FirebaseErrors.swift

```diff
@@ -20,24 +20,24 @@ enum FirebaseCommonError: Error {
     case unknown
 
     init(error: NSError) {
-        let firebaseError = FIRAuthErrorCode(rawValue: error.code)
+        let firebaseError = AuthErrorCode(rawValue: error.code)
 
         switch firebaseError {
-        case .errorCodeNetworkError?:
+        case .some(AuthErrorCode.networkError):
```

Comment:
Why don't you do it the same way it was done before? With the question mark at the end of the case?

Reply:
This returned error "Pattern cannot match values of type 'AuthErrorCode'"

Reply:
I just tried it and `case .networkError?:` works just fine, is there a problem with any of the other cases?

Reply:
It works now, maybe it was a compiler inconsistency. Anyway, I changed it back.